contributing-docs/11_provider_packages.rst

```diff
@@ -122,19 +122,21 @@ Local Development Release of a Specific Provider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When you develop a provider, you can release it locally and test it in your Airflow environment. This should
-be accomplished using breeze. Choose a suffix for the release such as "dev1" and run the breeze build for
+be accomplished using breeze. Choose a suffix for the release such as "+patch.asb.1" and run the breeze build for
 that provider. Remember Provider IDs use a dot ('.') for directory separators so the Provider ID for the
 Microsoft Azure provider is 'microsoft.azure'. This can be provided in the PACKAGE_LIST environment variable
 or passed on the command line.
 
-``export PACKAGE_LIST=microsoft.azure``
+```bash
+export PACKAGE_LIST=microsoft.azure
+````
 
 Then build the provider (you don't need to pass the package ID if you set the environment variable above):
 
 ```bash
 breeze release-management prepare-provider-packages \
-    --package-format both --version-suffix-for-pypi=dev1 \
-    --skip-tag-check microsoft.azure
+    --package-format both --version-suffix-for-local=+patch.asb.1 \
```

Comment:
It would be good to separate those two cases. I see a need to generate "dev1" or "rc" locally as well as "+patchwhatever" - for example when we are verifying packages for releases, PMC members prepare the same packages locally as they will be released - in order to test "binary reproducibiility"  - i.e. they verify if the package they produce locally is binary the same as the one that release manger produced (this is security feature of our release process). So it woudl be great if we show both cases AND explain a bit the difference betweeen the two - as it is not entirely obvious for someone who does not know the versioning PEP.

Reply:
I believe I addressed this in the code, but not in the docs yet.

Reply:
Looks goood. Indeed - just doc leaving old example would be great.

Reply:
I believe I added the old example you wanted back in.